tests/test_integration_default.py

```diff
@@ -271,8 +319,8 @@ def test_unused_snapshots_warning(stubs):
     assert result.ret == 0
 
 
-def test_unused_snapshots_ignored_if_not_targeted_when_targeting_using_dash_k(stubs):
```

Comment:
This was already covered by `test_collection` so repurposed to test for the node ids case